### DISCUSSION TITLE
Asset locations - checking assets in and out

### DISCUSSION THREAD

**Discussion by DonKoko** [Shelf-nu/shelf.nu#157] [Ideas]
The general idea is that we want users to be able to know where a certain asset is located. This is not an automated process(for now) but something that will be managed by the users of shelf.

## What do we need

- A way to create, edit and delete "Locations"(this is just a placeholder name)
- A way for being able to assign a location to an asset(via asset view)
- Being able to see the location in the asset page
- A location view that lists all assets within that location(maybe directly from there they can move assets to a different location)
- Automatic notes updates when an asset is moved to a different location
- QR scanning - when scanning a qr, an easy way to move the asset to a different location. Idea: When you land on asset page after scanning a qr code, get a contextual menu, with important actions, one of which can be "Change asset location"

**Comment by carlosvirreira** [comment]
is this something you can help with @jurrejansen ?
